islandora.module

```diff
@@ -1965,6 +1965,10 @@ function islandora_islandora_datastream_access($op, AbstractDatastream $datastre
   module_load_include('inc', 'islandora', 'includes/utilities');
   $result = islandora_namespace_accessible($datastream->parent->id) && user_access($op, $user);
```

Comment:
Actually... Instead of copypasta... could we just call `islandora_islandora_object_access()` explicitly here? As in:
```php
$result = islandora_islandora_object_access($op, $datastream->parent, $user);
```
should be equivalent?

Admittedly, would be a little strange to call an implementation of a hook specifically, but... should do the trick?... Dunno...

Reply:
That would break the way that IP embargo allows TN datastreams. 

I'd much rather just add back the logic that only lets you access a datastream if you can access an object - it could just go here: https://github.com/Islandora/islandora/blob/7.x/islandora.module#L1737-L1741